indico/modules/events/reminders/models/reminders.py

```diff
@@ -143,6 +151,40 @@ def all_recipients(self):
         recipients = set(self.recipients)
         if self.send_to_participants:
             recipients.update(reg.email for reg in Registration.get_all_for_event(self.event))
+
+        if self.send_to_speakers:
+            event_person_emails = (person_link.email for person_link in self.event.person_links)
+            # No contribution/sub contribution speakers present for lectures
+            if (self.event.type == 'lecture'):
```

Comment:
```suggestion
            if self.event.type == 'lecture':
```

Reply:
And I don't think this is needed as we always use the persons linked to the event itself. So you can just do this for what's in else right now:

```python
if self.event.type_ != EventType.lecture:
```

Reply:
Ah! This is a much better approach yes. I need to get into the habit of figuring out these patterns. I always tend to reuse this. Thanks for pointing out :)